bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -369,11 +369,14 @@ spec:
           fi
 
           # start nbctl daemon for caching
-          export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/tmp/ovnk-nbctl.pid \
+          export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/var/run/ovn/ovn-nbctl.pid \
             --detach \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
             --db "{{.OVN_NB_DB_LIST}}")
 
+           # REMOVEME once OVN path for control socket is fixed (right now uses /var/run/openvswitch)
+          ln -sf $OVN_NB_DAEMON /var/run/ovn/
```

Comment:
Hm... this will cause deployments to start failing immediately once OVN is fixed, right? ("failed to create symbolic link: File exists") You should make it only make the link if `$OVN_NB_DAEMON` is in /var/run/openvswitch.

Reply:
thanks will add || true. ln -sf to the same file will error so just need to avoid failing for this case